seed/analysis_pipelines/better/pipeline.py

```diff
@@ -368,69 +369,82 @@ def _process_results(self, analysis_id):
         ExtraDataColumnPath(
             'better_cost_savings_combined',
             'BETTER Potential Cost Savings (USD)',
+            1,
             'assessment.assessment_energy_use.cost_savings_combined'
         ),
         ExtraDataColumnPath(
             'better_energy_savings_combined',
             'BETTER Potential Energy Savings (kWh)',
+            1,
             'assessment.assessment_energy_use.energy_savings_combined'
         ),
         ExtraDataColumnPath(
             'better_ghg_reductions_combined',
-            'BETTER Potential GHG Emissions Reduction (kgCO2e)',
+            'BETTER Potential GHG Emissions Reduction (MTCO2e)',
```

Comment:
We are just using `t` for metric ton. That is what AT is using.

```suggestion
            'BETTER Potential GHG Emissions Reduction (t CO2e/yr)',
```